### PR TITLE
style: Allow having any year in the copyright header

### PR DESCRIPTION
**Pull request by grianbrcom**
Checkstyle was changed to check header with regex.

### CODE REVIEW

File: server/src/style/javaHeader.txt [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
- * which is available at https://www.eclipse.org/legal/epl-2.0/
- *
- * SPDX-License-Identifier: EPL-2.0
- *
- * Contributors:
- *    Broadcom, Inc. - initial API and implementation
- *
- */
+^/\*$
+^ \* Copyright \(c\) \d\d\d\d Broadcom\.$
```
Comment (asatklichov) [inline server/src/style/javaHeader.txt:2]: I think better to keep 2020 as for validation check. According to [US](https://rally1.rallydev.com/#/377806736628/teamboard?detail=%2Fuserstory%2F360326325276&view=1a5c8a9d-edbf-45aa-818c-215d6aa8f7af) we need to keep 2020 from now.

Comment (ishche) [reply server/src/style/javaHeader.txt:2]: This link is private. This decision should be available for people outside Broadcom.

Comment (asatklichov) [reply server/src/style/javaHeader.txt:2]: Easy, I will create a GitHub issue with info ;)

**Review by asatklichov — CHANGES_REQUESTED** [review]
This PR can be closed now.

**Comment by grianbrcom** [comment]
Yes, now I remember.

File: server/src/style/javaHeader.txt (hunk 1 above)
Comment (asatklichov) [reply server/src/style/javaHeader.txt:2]: d\d\d\d\  allows any 4 digit number - I guess we do need to manual check if year is actual (2021, ... ) right?

**Review by asatklichov — APPROVED** [review]
🏎️

**Review by temanbrcom — APPROVED** [review]

File: server/src/style/javaHeader.txt (hunk 1 above)
Comment (grianbrcom) [reply server/src/style/javaHeader.txt:2]: Yes, it allows any 4 digit number.
We can do something like `202\d`, but we have to change this in the future. With \d\d\d\d we are safe up to eight centuries =)

Comment (temanbrcom) [reply server/src/style/javaHeader.txt:2]: You are too much optimistic about the future of this project

Comment (ishche) [reply server/src/style/javaHeader.txt:2]: He is right to be so. It's the mainframe world.

**Pull request merged**